source/includes/investor-payments.md.erb

```diff
@@ -254,19 +252,21 @@ HTTP/1.1 200 OK
 Content-Type: application/json
 
 {
-  "bankReference" : "bankReference",
-  "accountNumber" : "accountNumber",
-  "sortCode" : "sortCode"
+  "bankReference" : "REF",
+  "accountName" : "John Smith",
+  "accountNumber" : "12345678",
+  "sortCode" : "123456"
 }
 ```
 ### Description
 Retrieve the bank details for depositing funds to the ISA account.
 ### Response
-| Name          | Type   | Description                            |
-| ------------- | ------ | -------------------------------------- |
-| accountNumber | string | The account number.                    |
-| sortCode      | string | The sort code.                         |
-| bankReference | string | The reference to use for the transfer. |
+| Name          | Type   | Description                                               |
+|---------------|--------|-----------------------------------------------------------|
+| accountName   | string | The account name. (can be used for Confirmation of Payee) |
+| accountNumber | string | The account number.                                       |
+| sortCode      | string | The sort code.                                            |
+| bankReference | string | The reference to use for the transfer.                    |
```

Comment:
technically not part of your change but!
wondering if we want to specify length for the bankReference + acceptable characters.